src/FunctionApps/python/GenerateCSVs/patient.py

```diff
@@ -0,0 +1,166 @@
+PATIENT_COLUMNS = [
+    "patient_hash",
+    "given_name",
+    "family_name",
+    "birthDate",
```

Comment:
$0.02: I'm not totally in love with the mixed case - can we pick underscores or camel?

Reply:
Dan had mentioned he wanted to keep the variable names as close to the FHIR spec as possible where it could apply. I think that's why you see some mixing (e.g. `birthDate`).

Reply:
As Brandon mentioned I just kept the names from FHIR. I figured that reducing on the number of different ways refer to birthday (especially when every STLT will have their own conventions) might make things a little easier down the road. Happy to pick names we like more though if others disagree.

Reply:
:shrug: I definitely get the point of keeping it in line with FHIR - in that case, let's make the new ones camel as well

Reply:
done